tools/clang/test/SemaHLSL/hlsl/workgraph/node_track_rw_input_sharing.hlsl

```diff
@@ -0,0 +1,31 @@
+// RUN: %dxc -T lib_6_8 -HV 2021 %s -verify
+
+struct RECORD
+{
+  uint a;
+  bool b;
+  uint3 grid : SV_DispatchGrid;
+};
+
+struct RECORD2
+{
+  uint a;
+  bool b;
+};
```

Comment:
It doesn't look like `RECORD` or `RECORD2` are used.  Should probably delete them.